README.rst

```diff
@@ -258,7 +258,7 @@ Webapp2
 
 .. code:: python
 
-    from opencensus.trace.tracer import webapp2_tracer
+    from opencensus.trace.tracers import webapp2_tracer
```

Comment:
I can't find the webapp2 files, are you going to add them to the 'ext' directory in this PR or a later one?

Reply:
Deleted it for now, probably add it in later PR.